plugins/modules/ecs_service.py

```diff
@@ -218,6 +218,12 @@
         required: false
         choices: ["DAEMON", "REPLICA"]
         type: str
+    wait:
+        description:
+          - Whether or not to wait for the desired state.
```

Comment:
```suggestion
          - Whether or not to wait for the desired state.
          - Waits only when I(state=absent).
```
 Something like that?

Reply:
Or should it be `- Waits only when I(state) is C(absent)`.. probably doesnt matter?